go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.9.0
 	github.com/teekennedy/goldmark-markdown v0.3.0
-	github.com/terraform-providers/terraform-provider-random/randomshim v0.0.0-00010101000000-000000000000
```

Comment:
Will actual random provider build with this fine? It has its own dependency on this, and its own linker trick.